src/Validators/Validators.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Validators;
+
+use App\Interfaces\ValidationInterface;
+use Exception;
+
+class Validators implements ValidationInterface
+{
+
+    public static function ValidateNewPost($unvalidated): array

```

Comment:
methods should be camelCase, also unvalidated here isn't entirely clear what that means to be, if you implement it like this, it can also be type hinted to only accept the object that you are creating and sending into it too.